01-Introducing-Molecular-Dynamics/01b-Rigid-Body-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import rowan
</pre><p>I don't think you ever use <code>rowan</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>